src/Translator/translations/sk.php

```diff
@@ -0,0 +1,188 @@
+	'step-day-of-month' => 'každý {step}. deň v mesiaci {part}',
+	'step-month' => 'každý {step}. mesiac {part}',
+	'range-minute' => 'od {left} do {right}',
+	'range-minute-named' => 'každú minútu od {left} do {right}',
+	'range-hour' => 'od {left} do {right}',
+	'range-hour-named' => 'každú hodinu od {left} do {right}',
+	'range-day-of-week' => 'od {left} do {right}',
+	'range-day-of-week-named' => 'každý deň v týždni od {left} do {right}',
+	'range-day-of-month' => 'od {left} do {right}',
+	'range-day-of-month-named' => 'každý deň v mesiaci od {left} do {right}',
+	'range-month' => 'od {left} do {right}',
+	'range-month-named' => 'každý mesiac od {left} do {right}',
+	'second' => '{second, plural,
+      one {každú sekundu}
+      few {každé # sekundy}
+      other {každých # sekúnd}
+    }',
+	'every-minute' => 'každú minútu',
+	'before-minute' => '',
+	'minute' => '{minute}',
+	'minute-named' => 'v minúte {minute}',
+	'before-hour' => ' ',
+	'hour' => '{hour}',
+	'hour-named' => 'v hodine {hour}',
+	'between-day-of-month-and-week' => ' a',
+	'before-day-of-week' => '{dayNumber, select,
+	  4 { vo }
+      other { v }
+	}',
+	'day-of-week' => '{context, select,
+      step {{dayNumber, select,
+        1 {pondelka}
+        2 {utorka}
+        3 {stredy}
+        4 {štvrtka}
+        5 {piatka}
+        6 {soboty}
+        7 {nedele}
+        other {{dayNumber} - unknown}
+      }}
+      range {{dayNumber, select,
+        1 {pondelka}
+        2 {utorka}
+        3 {stredy}
+        4 {štvrtka}
+        5 {piatka}
+        6 {soboty}
+        7 {nedele}
+        other {{dayNumber} - unknown}
+      }}
+      other {{dayNumber, select,
+        1 {pondelok}
+        2 {utorok}
+        3 {stredu}
+        4 {štvrtok}
+        5 {piatok}
+        6 {sobotu}
+        7 {nedeľu}
+        other {{dayNumber} - unknown}
+      }}
+    }',
+	'day-of-week-nth' => '{nth}. {day}',
+	'day-of-week-last' => '{context, select,
+      range {{dayNumber, select,
+        1 {posledného pondelka}
+        2 {posledného utorka}
+        3 {poslednej stredy}
+        4 {posledného štvrtka}
+        5 {posledného piatku}
+        6 {poslednej soboty}
+        7 {poslednej nedele}
+        other {{day} - unknown}
+      }}
+      other {{dayNumber, select,
+        1 {posledný}
+		2 {posledný}
+		3 {poslednú}
+		4 {posledný}
+		5 {posledný}
+		6 {poslednú}
+		7 {poslednú}
+		other {unknown}
+      } {day}}
+    }',
+	'before-day-of-month' => ' ',
+	'day-of-month' => '{day}',
+	'day-of-month-named' => 'dňa v mesiaci {day}',
+	'day-of-month-last-day' => '{context, select,
+      range {posledného dňa v mesiaci}
+      other {posledný deň v mesiaci}
+    }',
+	'day-of-month-last-weekday' => '{context, select,
+      range {posledného pracovného dňa}
+      other {v posledný pracovný deň}
+    }',
+	'day-of-month-nearest-weekday' => '{context, select,
+      range {pracovného dňa najbližšieho k {day}.}
+      other {pracovného dňa najbližšieho k {day}.}
+    }',
```

Comment:
You don't have to use select when all values are the same

```suggestion
	'day-of-month-nearest-weekday' => '{pracovného dňa najbližšieho k {day}.',
```